quickwit/quickwit-serve/src/node_info_handler.rs

```diff
@@ -23,16 +23,24 @@ use quickwit_config::QuickwitConfig;
 use serde_json::json;
 use warp::{Filter, Rejection};
 
+use crate::elastic_search_api::es_compat_info_handler;
 use crate::{with_arg, BuildInfo, RuntimeInfo};
 
+#[derive(utoipa::OpenApi)]
+#[openapi(paths(node_version_handler, node_config_handler,))]
+pub struct NodeInfoApi;
+
 pub fn node_info_handler(
     build_info: &'static BuildInfo,
     runtime_info: &'static RuntimeInfo,
     config: Arc<QuickwitConfig>,
 ) -> impl Filter<Extract = (impl warp::Reply,), Error = Rejection> + Clone {
-    node_version_handler(build_info, runtime_info).or(node_config_handler(config))
+    node_version_handler(build_info, runtime_info)
+        .or(node_config_handler(config.clone()))
+        .or(es_compat_info_handler(build_info, config))
```

Comment:
This is not the place to put ES handler right?

Reply:
I think these are the top-level handlers for cluster info. That's why it is built in ES folder but mounted here. functionally they belong to the same group. But I can just mount it there if you think this is more appropriate to you.

Reply:
But we want to keep **all** elasticsearch handlers mounted at the same place, under the `_elastic` path, that's the logic currently coded in there.

Reply:
I have changed to mount all elastic-related endpoints from the same location.